### PR TITLE
[ISSUE #3794] websocket datasync can chose allow origin to avoid CSRF

### PR DESCRIPTION
**Pull request by 847850277**
Dear Community:
I have finished  [#3794](https://github.com/apache/shenyu/issues/3794)
please review code.

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by yu199195** [comment]
hi，pls check ci

File: shenyu-bootstrap/src/main/resources/application.yml [hunk 1]
```diff
@@ -181,6 +181,7 @@ shenyu:
   sync:
     websocket:
       urls: ws://localhost:9095/websocket
+      allowOrigin: ws://localhost:9195
```
Comment (qicz) [inline shenyu-bootstrap/src/main/resources/application.yml:184]: missing 's' ?

File: shenyu-integrated-test/shenyu-integrated-test-alibaba-dubbo/src/main/resources/application-local.yml [hunk 2]
```diff
@@ -40,6 +40,7 @@ shenyu:
   sync:
     websocket:
       urls: ws://localhost:9095/websocket
+      allowOrigin: ws://localhost:9195
```
Comment (qicz) [inline shenyu-integrated-test/shenyu-integrated-test-alibaba-dubbo/src/main/resources/application-local.yml:43]: missing 's' ?

File: shenyu-integrated-test/shenyu-integrated-test-apache-dubbo/src/main/resources/application-local.yml [hunk 3]
```diff
@@ -36,6 +36,7 @@ shenyu:
   sync:
     websocket:
       urls: ws://localhost:9095/websocket
+      allowOrigin: ws://localhost:9195
```
Comment (qicz) [inline shenyu-integrated-test/shenyu-integrated-test-apache-dubbo/src/main/resources/application-local.yml:39]: missing 's' ?

File: shenyu-integrated-test/shenyu-integrated-test-combination/src/main/resources/application-local.yml [hunk 4]
```diff
@@ -36,6 +36,7 @@ shenyu:
   sync:
     websocket:
       urls: ws://localhost:9095/websocket
+      allowOrigin: ws://localhost:9195
```
Comment (qicz) [inline shenyu-integrated-test/shenyu-integrated-test-combination/src/main/resources/application-local.yml:39]: missing 's' ?

**Comment by yu199195** [comment]
@847850277  hi， Can you tell us exactly what allow origin does?

**Comment by 847850277** [comment]
@yu199195  to avoid CSRF, In the case of Internet environment if someone knows admin websocket url,he can connect to ShenYu admin.Then forge the attack.
of course , this config is an option , this just match allow origin , if you dont't want , you can chose old mode.

**Review by yu199195 — APPROVED** [review]

**Comment by yu199195** [comment]
Thanks， it merged~

**Pull request merged**